iati/core/tests/test_rulesets.py

```diff
@@ -287,22 +309,36 @@ def test_is_valid_for_raises_error_when_passed_an_etree(self, rule_basic_init):
         with pytest.raises(AttributeError):
             rule_basic_init.is_valid_for(iati.core.resources.load_as_tree(iati.core.resources.get_test_data_path('valid_atleastone')))
 
-    def test_empty_xpath_base_rule_init_normalised_paths(self, rule_empty_xpath_base):
-        """Check that a Rule with an empty xpath_base can be instantiated correctly with normalised `paths`.
+    def test_multiple_valid_context_matches_is_valid_for(self, valid_multiple_context, valid_nest_case, rule_constructor, valid_dataset):
+        """Check Rule returns expected result when checking multiple contexts."""
+        rule = rule_constructor(valid_multiple_context, valid_nest_case)
+        assert rule.is_valid_for(valid_dataset)
+
+    def test_multiple_valid_context_matches_is_invalid_for(self, valid_multiple_context, invalid_nest_case, rule_constructor, invalid_dataset):
+        """Check Rule returns expected result when checking multiple contexts."""
+        rule = rule_constructor(valid_multiple_context, invalid_nest_case)
+        assert not rule.is_valid_for(invalid_dataset)
+
+    def test_condition_case_is_True(self, valid_condition_rule, valid_dataset):
+        """Check that if a condition is `True`, the rule returns None."""
```

Comment:
Some explanation about skipping would be useful in the docstring here.

Reply:
done